diff --git a/build/cjs/src/gaxios.d.ts b/build/cjs/src/gaxios.d.ts
index dfa3a671f9b14c5be6db633d03e9735c04bd7dc7..b957c3d1fb06c892e0d9661ab26baafcd3c6c87c 100644
--- a/build/cjs/src/gaxios.d.ts
+++ b/build/cjs/src/gaxios.d.ts
@@ -10,7 +10,7 @@ import { GaxiosInterceptorManager } from './interceptor.js';
  * compatible with the `fetch` API.
  */
 interface FetchCompliance {
-    fetch: typeof fetch;
+    fetch: (...args: Parameters<typeof fetch>) => ReturnType<typeof fetch>;
 }
 export declare class Gaxios implements FetchCompliance {
     #private;
diff --git a/build/esm/src/gaxios.d.ts b/build/esm/src/gaxios.d.ts
index dfa3a671f9b14c5be6db633d03e9735c04bd7dc7..b957c3d1fb06c892e0d9661ab26baafcd3c6c87c 100644
--- a/build/esm/src/gaxios.d.ts
+++ b/build/esm/src/gaxios.d.ts
@@ -10,7 +10,7 @@ import { GaxiosInterceptorManager } from './interceptor.js';
  * compatible with the `fetch` API.
  */
 interface FetchCompliance {
-    fetch: typeof fetch;
+    fetch: (...args: Parameters<typeof fetch>) => ReturnType<typeof fetch>;
 }
 export declare class Gaxios implements FetchCompliance {
     #private;
